src/nifax/control/view/event/btn/Btn_deleteGralAction.java

```diff
@@ -0,0 +1,35 @@
+package nifax.control.view.event.btn;
+
+import com.sun.glass.events.KeyEvent;
+import java.awt.event.ActionEvent;
+import javax.swing.AbstractAction;
+import static javax.swing.Action.MNEMONIC_KEY;
+import static javax.swing.Action.SHORT_DESCRIPTION;
+import nifax.control.controller.TableGralController;
+import nifax.control.view.panel.PanelGeneralAdmin;
+
+/**
+ *
+ * @author NB
+ */
+public class Btn_deleteGralAction extends AbstractAction {
+
+    private final PanelGeneralAdmin panelGeneralAdmin;
+
+    public Btn_deleteGralAction(PanelGeneralAdmin panelGeneralAdmin) {
+        super("Eliminar");
+        this.panelGeneralAdmin = panelGeneralAdmin;
+        putValue(SHORT_DESCRIPTION, "Presionando este boton se eliminaran los elementos seleccionados en la tabla");
+        putValue(MNEMONIC_KEY, KeyEvent.VK_D);
+    }
+
+    @Override
+    public void actionPerformed(ActionEvent e) {
+
+        String titleTab = panelGeneralAdmin.getTbp_gral().getTitleAt(0);
```

Comment:
Allways when you use indexes you need to clarify what is indexing that value throught a comment or it will be better if you use a constant or enumerator